src/objects/media/media.stories.mdx

```diff
@@ -175,3 +175,40 @@ By adding the `o-media--jaunty` class, the media object will have a fun little r
     {(args) => imageDemo(args)}
   </Story>
 </Canvas>
+
+## Align Start
+
+By adding the `o-media--align-start` class, the media object will use `align-items: start`.
+This can come in handy when the content is vertically shorter than the object
+and there are multiple Media objects side-by-side with varying content.
+It provides a way to visually align the multiple Media objects.
+
+<Canvas>
+  <Story
+    name="Align Default"
+    args={{
+      text: 'Default alignment',
+    }}
+    parameters={{
+      docs: {
+        transformSource: imageDemoTransformSource,
+      },
+    }}
+  >
+    {(args) => imageDemo(args)}
+  </Story>
+  <Story
+    name="Align Start"
+    args={{
+      align_start: true,
+      text: 'Align start alignment',
+    }}
+    parameters={{
+      docs: {
+        transformSource: imageDemoTransformSource,
+      },
+    }}
+  >
+    {(args) => imageDemo(args)}
+  </Story>
+</Canvas>
```

Comment:
Should we add a "Template Properties" section now that we have several?

Reply:
@tylersticka Good idea!

It [this](https://github.com/cloudfour/cloudfour.com-patterns/pull/1791/commits/3d8522e5f0b8c4956a12ebc10cce206cbb21c817) what you had in mind? It's still WIP (I'll have to finish tomorrow), but I am finding it a bit odd I need to connect the `<ArgsTable>` to a specific story. Should I add an `<ArgsTable>` to all nine individual stories? 🤔 

<img width="1021" alt="Screen Shot 2022-05-12 at 2 28 08 PM" src="https://user-images.githubusercontent.com/459757/168171436-63adc3dc-ac85-4eb5-a35d-207e669ea3f3.png">

Reply:
@gerardo-rodriguez `ArgsTable` is a good choice if the story's properties are the same as the template's properties. In this case, I'm skeptical that `imgSrc` and `text` are template properties?

If in doubt, a simple list will suffice. Here's an example: https://v-next--cloudfour-patterns.netlify.app/?path=/docs/components-alert--basic#template-properties

Reply:
[Added](https://github.com/cloudfour/cloudfour.com-patterns/pull/1791/commits/0a9073b95fd889f127def9245d2252bdb547b5b7)!

<img width="852" alt="Screen Shot 2022-05-13 at 5 34 24 AM" src="https://user-images.githubusercontent.com/459757/168284116-09790736-8ec1-42bb-945f-7e1680530142.png">